subprojects/gradle/ci-logger/src/main/kotlin/com/avito/utils/logging/CILoggingFormatter.kt

```diff
@@ -0,0 +1,39 @@
+package com.avito.utils.logging
+
+import java.io.Serializable
+import java.text.DateFormat
+import java.text.SimpleDateFormat
+import java.util.Date
+
+interface CILoggingFormatter : Serializable {
+
+    fun format(message: String): String
+    fun child(tag: String): CILoggingFormatter
+}
+
+internal object NothingLoggingFormatter : CILoggingFormatter {
```

Comment:
"Nothing Logging" - it is strange. Formatters don't know about further operations - destinations, real logging, ...
I propose "NoOpFormatter" or similar terms.

Reply:
I will do better. We could delete this